openquake/calculators/postproc/compute_rtgm.py

```diff
@@ -326,6 +335,83 @@ def get_asce41(dstore, mce, facts):
             'BSE1N_S1': BSE1N_S1}
 
 
+def plot_meanHCs_afe_RTGM(imls, AFE, UHGM_RP, afe_RP, RTGM, afe_RTGM,
+                          imt_list):
+    plt.figure(figsize=(12, 9))
+    plt.rcParams.update({'font.size': 15})
+    for i, imt in enumerate(imt_list):
+        plt.loglog(imls[i], AFE[i], label='Mean ' + imt, linewidth=3)
+        plt.loglog([np.min(imls[i]), UHGM_RP[i]], [afe_RP, afe_RP], 'k--',
+                   linewidth=2)
+        plt.loglog([UHGM_RP[i], UHGM_RP[i]], [0, afe_RP], 'k--', linewidth=2)
+        plt.loglog([UHGM_RP[i]], [afe_RP], 'ko', linewidth=2)
+        plt.loglog([np.min(imls[i]), RTGM[i]], [afe_RTGM[i], afe_RTGM[i]],
+                   'r--', linewidth=2)
+        plt.loglog([RTGM[i], RTGM[i]], [0, afe_RTGM[i]], 'r--', linewidth=2)
+        plt.loglog([RTGM[i]], [afe_RTGM[i]], 'ro', linewidth=2)
+    plt.grid('both')
+    plt.legend(fontsize=16)
+    plt.xlabel('Acceleration (g)', fontsize=20)
+    plt.ylabel('Annual frequency of exceedance', fontsize=20)
+    plt.legend(loc="best", fontsize='16')
+    plt.ylim([10E-6, 1])
+    plt.xlim([np.min(imls[i]), 4])
+    bio = io.BytesIO()
+    plt.savefig(bio, format='png', bbox_inches='tight')
+    return Image.open(bio)
+
+
+def _poe2afe(poes, window):
+    # convert from poe to rates of exceedance
+    afe = [-np.log(1-p)/window for p in poes]
+    return afe
```

Comment:
This is already available and called to_rates; also, since we have already the rates it is probably not needed.

Reply:
Done